ml_git/repository.py

```diff
@@ -1044,7 +1044,8 @@ def create(self, kwargs):
                 log.info(output_messages['INFO_UNZIPPING_FILES'], CLASS_NAME=REPOSITORY_CLASS_NAME)
                 data_path = os.path.join(get_root_path(), repo_type, kwargs['entity_dir'], artifact_name, 'data')
                 unzip_files_in_directory(data_path)
-            log.info(output_messages['INFO_PROJECT_CREATED'], CLASS_NAME=REPOSITORY_CLASS_NAME)
+            message_key = 'INFO_%s_CREATED' % self.__repo_type.upper()
```

Comment:
Please use `str.format()` to formatting,

```suggestion
            message_key = 'INFO_ENTITY_CREATED'.format(self.__repo_type)
```

Reply:
changed to use str.format () https://github.com/HPInc/ml-git/pull/134/commits/313ab8174b3987a90e04f585bc0a9ce7e1e31d9b

Reply:
Please apply this change in the other necessary places.